automated install/basic-install.sh

```diff
@@ -368,13 +368,12 @@ package_manager_detect() {
 
         # If the host OS is centos (or a derivative), epel is required for lighttpd
         if ! grep -qiE 'fedora|fedberry' /etc/redhat-release; then
-            # Check current CentOS major release version
-            CURRENT_CENTOS_VERSION=$(grep -oP '(?<= )[0-9]+(?=\.?)' /etc/redhat-release)
             if rpm -qa | grep -qi 'epel'; then
                 printf "  %b EPEL repository already installed\\n" "${TICK}"
             else
-                # CentOS requires the EPEL repository to gain access to Fedora packages
-                EPEL_PKG="https://dl.fedoraproject.org/pub/epel/epel-release-latest-${CURRENT_CENTOS_VERSION}.noarch.rpm"
+                # EPEL not already installed, add it based on the release version
+                RELEASE_VERSION=$(grep -oP '(?<= )[0-9]+(?=\.?)' /etc/redhat-release)
```

Comment:
```suggestion
                local RELEASE_VERSION
                RELEASE_VERSION=$(grep -oP '(?<= )[0-9]+(?=\.?)' /etc/redhat-release)
```

Reply:
Then also probably should do EPEL_PKG, too

Reply:
Sure. `RELEASE_VERSION` only has a higher change to be re-used somewhere else in the code for something else.

Reply:
Yeah, that's a fair point. But, for consistency I'll go with both.

Reply:
FP'd

Reply:
My suggestion is to use `EPEL_RELEASE` or `RH_RELEASE` since we get its value from `/etc/redhat-release`.